src/Builder/FormContractor.php

```diff
@@ -14,176 +14,39 @@
 namespace Sonata\DoctrineMongoDBAdminBundle\Builder;
 
 use Doctrine\ODM\MongoDB\Mapping\ClassMetadata;
-use Sonata\AdminBundle\Admin\AdminInterface;
-use Sonata\AdminBundle\Admin\FieldDescriptionInterface;
-use Sonata\AdminBundle\Builder\FormContractorInterface;
-use Sonata\AdminBundle\Form\Type\AdminType;
-use Sonata\AdminBundle\Form\Type\ModelAutocompleteType;
-use Sonata\AdminBundle\Form\Type\ModelHiddenType;
-use Sonata\AdminBundle\Form\Type\ModelListType;
-use Sonata\AdminBundle\Form\Type\ModelType;
-use Sonata\Form\Type\CollectionType;
-use Symfony\Component\Form\Extension\Core\Type\FormType;
-use Symfony\Component\Form\FormBuilderInterface;
-use Symfony\Component\Form\FormFactoryInterface;
+use Sonata\AdminBundle\Builder\AbstractFormContractor;
+use Sonata\AdminBundle\FieldDescription\FieldDescriptionInterface;
 
-final class FormContractor implements FormContractorInterface
+final class FormContractor extends AbstractFormContractor
```

Comment:
@franmomu Do we even need to create a FormContractor in persistence bundle ?

The AbstractFormContractor could be a FormContractor, no ?

Reply:
I don't think we need it anymore and probably we could move other builders there.